### ISSUE TITLE
Does this library still receive maintenance?

### ISSUE THREAD

**Issue by itsManeka** [node-facebook/facebook-node-sdk#192]
Does this library still receive maintenance?

**Comment by dantman** [comment]
No, I'm not sure if FB is still using the old graph API after creating GraphQL.

**Comment by itsManeka** [comment]
lol I'm using it to post on my page and it works, but npm notifies me of vulnerabilities. Any current package recommendations?

**Comment by dantman** [comment]
No, I would hope that after all these years Facebook would publish some official documentation on how to use their API.
